cirrus/server/Dockerfile

```diff
@@ -46,5 +54,11 @@ ENV PYTHONPATH=$PYTHONPATH:/application-services
 # Copy all other files into the container's working directory
 COPY . .
 
+# Change ownership of the working directory to the non-root user
+RUN chown -R $USERNAME:$USERNAME /cirrus
+
+# Switch to non-root user
+USER $USERNAME
+
```

Comment:
Should you switch to the non-root user? This would make this user the default

Reply:
sorry I am not sure :(, do you mean why we need to switch?

Reply:
Oh I see, I missed the variable name!